examples/framework/preferences/minimalEditor/schemas/auxiliary.js

```diff
@@ -0,0 +1,58 @@
+ /*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+var minEditor = minEditor || {};
```

Comment:
fluid.registerNamespace("minEditor") - but actually unnecessary since we make no use of the global, and we issue a defaults within it anyway